index.html

```diff
@@ -0,0 +1,578 @@
+  </figcaption>  
+  <code class=anatomy>
+    <span class=pink style='--depth: 2'>https://example.com/<span>exampleSpec<aside>name</aside></span><aside>
+      identity
+    </aside></span>/<span style='--depth: 2' class=green>v1.0<aside>version</aside></span><span class=grey>?key=val&k2=v2#fragment<aside>ignored</aside></span>
+  </code>
+</figure>
+
+### Why is versioning in the URL, not a directive argument?
+
+The version is in the URL because when a human reader visits the URL, we would like them to be taken to the documentation for the *version of the specification in use by this document*. Many text editors will turn URLs into hyperlinks, and it's highly desirable that clicking the link takes the user to the correct version of the docs. Putting the version information in a separate argument to the `@core` directive would require this.
+
+## Prefixing
+Specifications meant to be referenced via `@core` **should** prefix any schema elements they introduce. The prefix:
+1. **must** be a string of characters valid within GraphQL typenames, with the exception of two underscores (`"__"`)
+2. **must** match the default name of the spec, as derived from the spec's URL.
+
+With the exception of their root directive, specs **should** define any new schema elements with names that start with `$name__`. The root directive can simply be named with the name of the spec.
+
+```graphql
+#::[Using the `@core` directive with a spec's default name]
+schema @core(using: "https://spec.example.com/example/v1.0") {
+  query: Query
+}
+
+type User {
+  name: String @example(data: ITEM)
+}
+
+# An enum used to provide structured data to the example spec.
+# It is prefixed with the name of the spec.
+enum example__Data {
+  ITEM
+}
+
+directive @example(data: example__Data) on FIELD
+```
+
+The language of the spec **should** specify the default prefix and **should** identify locations where the prefix is used. The prefix **should not** be elided within documentation; definitions of schema elements provided within the spec **must** include the default prefix.
+
+### Elements which should be prefixed
+Specs **should** attach their prefix to:
+- the names of any object types, interfaces, unions, enums, or input object types defined in the spec
+- the names of any fields the spec introduces on *foreign types* defined in a different spec
+- the names of any arguments the spec introduces on *foreign directives and fields* defined in a different spec
+- the names of any directives introduced in the spec
+  - specs may introduce one **root directive**, which carries *only* the prefix and no underscore.
+
+<view-of src=prefixing.graphql
+  title='Prefixing examples'
+></view-of>
+
+## Versioning
+
+Specs are versioned with a **subset** of a [Semantic Version Number](https://semver.org/spec/v2.0.0.html) containing only the `major` and `minor` parts. Thus, specifications **should** provide a version of the form `‹major›.‹minor›`, where both integers >= 0.
```

Comment:
A slight drive-by comment here (I'm going to do another round of review of this spec next week, but until then just wanted to chime in here a bit): I'm of the opinion that breaking changes to a spec should be indicated by a major version bump instead of an entirely new spec. I'd like to write more about why I'd prefer that (it relates to the versioning doc I'm writing up), but really this comment is just here to make sure we don't decide on a change early and start propagating the change out to implementation. I'll edit this comment next week to explain more about my reasoning here, but if it's blocking let me know and I can prioritize this sooner.

Reply:
While I am super curious to hear your perspective, I don't get the impression that I have won anybody over with my view so there's probably not too much to worry about here :)